### PR TITLE
iscsistart: Fix iscsistart timeout during login session

### PR DESCRIPTION
**Pull request by jkhasdev-ora**
When iscsistart places a login request during boot, if it gets timeout
and response receive after it. It triggers "connection1:0: detected conn
error (1011)", since in userspace nobody is listening.

On OL6, it has been observed where iscsid is not part of initramfs.

During boot, iscsistart forks and child process used as placing a request whereas
parent wait in event loop.

In the child, login_seesion places login request, and request gets
timed-out then parent also exit. If it receive response for timed-out
request and during that time no parent is listening to it then it will
error as 1011.

This patch adds an additional option to increase tmo for login request
timeout. This patch has been tested some intesive workload by increase it as 6x
and helps resolve the issue.

Signed-off-by: Jitendra Khasdev <jitendra.khasdev@oracle.com>

### CODE REVIEW

**Comment by gonzoleeman** [comment]
I closed the other ("[OLD]") pull request, and now this one seem  broken. When I try to look at the diffs, it says HTML 404 error page not found?

File: usr/iscsistart.c [hunk 1]
```diff
@@ -254,7 +258,7 @@ static int login_session(struct node_rec *rec)
 	 * login.
 	 */
 	for (msec = 50; msec <= 15000; msec <<= 1) {
-		int tmo = ISCSID_REQ_TIMEOUT * 10;
+		int tmo = ISCSID_REQ_TIMEOUT * login_req_timeout;
```
Comment (gonzoleeman) [inline usr/iscsistart.c:261]: I don't like multiplying ISCSID_REQ_TIMEOUT times the login_req_timeout. It just so happens that currently ISCSID_REQ_TIMEOUT is one second, but what if it changes to 5 seconds? Then your iscsistart timout would multiply times 5! Can you just login_req_timeout not depend on ISCSID_REQ_TIMEOUT?

**Review by gonzoleeman — CHANGES_REQUESTED** [review]
The issistart man page also needs to be updated. Please add that.

**Comment by gonzoleeman** [comment]
One last small change requested. Can you please update your branch so I can pull your request?

**Comment by mikechristie** [comment]
Hey Lee, jkhasdev-ora is no longer at oracle and is free from iscsi :)

Me on the other hand.... is hopefully temporarily helping out :)

This patch is wrong. I'm going to close this and will open a new PR.

**Pull request closed without merging**